sample_metadata/parser/cloudhelper.py

```diff
@@ -97,7 +97,12 @@ async def file_contents(self, filename) -> str | None:
 
     async def file_exists(self, filename: str) -> bool:
         """Determines whether a file exists"""
-        return AnyPath(self.file_path(filename)).exists()
+        path = self.file_path(filename)
+        if path.startswith('gs://'):
```

Comment:
Maybe add a comment here why we have a special case for GCS?

Reply:
Good idea, added a comment :)